src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsregisterstonomisupdate/health/QueueHealth.kt

```diff
@@ -0,0 +1,102 @@
+package uk.gov.justice.digital.hmpps.hmppsregisterstonomisupdate.health
```

Comment:
There is a unit test for this as well that can be copied

Reply:
ok - will ensure that gets pushed thru on next submit